lib/data/use_case/anime/get_top_anime.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:riverpod_annotation/riverpod_annotation.dart';
 
-import '../../../presentation/feature/news/models/top_anime_item_model.dart';
+import '../../../presentation/feature/record_lecture/models/top_anime_item_model.dart';
```

Comment:
Even though it is `get_top_anime.dart` this has been changed to `record_lecture`